version.py

```diff
@@ -53,7 +53,7 @@ def version(ip):
                     return None
         try:
             tn.read_until(flag)
-            tn.write('opkg list | grep ^cgminer | cut -c11-\n')
+            tn.write('cat /etc/avalon_version\n')
```

Comment:
@JianXiongZ NOTE: If you upgrade the cgminer package, then the version should not match the /etc/avalon_version.